src/embezzler.ts

```diff
@@ -106,13 +106,30 @@ export class EmbezzlerFight {
 
   /**
    * This is the class that creates all the different ways to fight embezzlers
-   * @classdesc Something goes here
+   * @classdesc Embezzler Fight enc
    * @prop {string} name The name of the source of this fight, primarily used to identify special cases.
-   * @prop {() => boolean} available Returns whether or not we can do this fight right now.
-   * @prop {() => number} potential Returns the number of embezzlers we expect to be able to fight from this source.
-   * @prop {(options: EmbezzlerFightOptions) => void} run This runs the combat, optionally using the provided location and macro. Location is used only by draggable fights.
-   * @prop {EmbezzlerFightConfigOptions} options configuration options for this fight
-   *
+   * @prop {() => boolean} available Returns whether or not we can do this fight right now (this may change later in the day).
+   * @prop {() => number} potential Returns the number of embezzlers we expect to be able to fight from this source given the current state of hte character
+   *  This is used when computing turns for buffs, so it should be as accurate as possible to the number of KGE we will fight
+   * @prop {(options: EmbezzlerFightRunOptions) => void} execute This runs the combat, optionally using the provided location and macro. Location is used only by draggable fights.
+   *  This is the meat of each fight. How do you initialize the fight? Are there any special considerations?
+   * @prop {EmbezzlerFightConfigOptions} options configuration options for this fight. see EmbezzlerFightConfigOptions for full details of all available options
+   * @example
+   * // suppose that we wanted to add a fight that will use print screens repeatedly, as long as we have them in our inventory
```

Comment:
so, I like the rework of this docstring quite a lot. I think it's a very good enumeration. my only small quibble is that this print screen example isn't currently in garbo, so if there are future changes we have to remember to update this example to match new changes (and an enterprising code-sniffer wouldn't be able to just compare the example with the actual embezzlerFight). this is not a huge issue though and i would put this more as a minor quibble/callout vs a real problem. i think this is a great example, it's just funny it isn't in embezzler.ts in general.